README.md

```diff
@@ -128,284 +43,8 @@
-  Use the indentation shown below for the `with` special form:
-  <sup>[[link](#with-indentation)]</sup>
-
-  ```elixir
-  with {year, ""} <- Integer.parse(year),
-       {month, ""} <- Integer.parse(month),
-       {day, ""} <- Integer.parse(day) do
-    new(year, month, day)
-  else
-    _ ->
-      {:error, :invalid_format}
-  end
-  ```
-
-  Always use the indentation above if there's an `else` option. If there isn't, the following indentation works as well:
-
-  ```elixir
-  with {:ok, date} <- Calendar.ISO.date(year, month, day),
-       {:ok, time} <- Time.new(hour, minute, second, microsecond),
-       do: new(date, time)
-  ```
-
-* <a name="for-indentation"></a>
-  Use the indentation shown below for the `for` special form:
-  <sup>[[link](#for-indentation)]</sup>
-
-  ```elixir
-  for {alias, _module} <- aliases_from_env(server),
-      [name] = Module.split(alias),
-      starts_with?(name, hint),
-      into: [] do
-    %{kind: :module, type: :alias, name: name}
-  end
-  ```
-
-  If the body of the `do` block is short, the following indentation works as well:
-
-  ```elixir
-  for partition <- 0..(partitions - 1),
-      pair <- safe_lookup(registry, partition, key),
-      into: [],
-      do: pair
-  ```
-
-* <a name="expression-group-alignment"></a>
-  Avoid aligning expression groups:
-  <sup>[[link](#expression-group-alignment)]</sup>
-
-  ```elixir
-  # Bad
-  module = env.module
-  arity  = length(args)
-
-  def inspect(false), do: "false"
-  def inspect(true),  do: "true"
-  def inspect(nil),   do: "nil"
-
-  # Good
-  module = env.module
-  arity = length(args)
-
-  def inspect(false), do: "false"
-  def inspect(true), do: "true"
-  def inspect(nil), do: "nil"
-  ```
-
-  The same non-alignment rule applies to `<-` and `->` clauses as well.
-
-* <a name="pipeline-indentation"></a>
-  Use a single level of indentation for multi-line pipelines.
-  <sup>[[link](#pipeline-indentation)]</sup>
-
-  ```elixir
-  input
-  |> String.strip()
-  |> String.downcase()
-  |> String.slice(1, 3)
-  ```
-
-### Term representation
-
-* <a name="underscores-in-numerics"></a>
-  Add underscores to decimal literals that have six or more digits.
-  <sup>[[link](#underscores-in-numerics)]</sup>
-
-  ```elixir
-  # Bad
-  num = 1000000
-  num = 1_500
-
-  # Good
-  num = 1_000_000
-  num = 1500
-  ```
-
-* <a name="hex-literals"></a>
-  Use uppercase letters when using hex literals.
-  <sup>[[link](#hex-literals)]</sup>
-
-  ```elixir
-  # Bad
-  <<0xef, 0xbb, 0xbf>>
-
-  # Good
-  <<0xEF, 0xBB, 0xBF>>
-  ```
-
-* <a name="quotes-around-atoms"></a>
-  When using atom literals that need to be quoted because they contain characters that are invalid in atoms (such as `:"foo-bar"`), use double quotes around the atom name:
-  <sup>[[link](#quotes-around-atoms)]</sup>
-
-  ```elixir
-  # Bad
-  :'foo-bar'
-  :'atom number #{index}'
-
-  # Good
-  :"foo-bar"
-  :"atom number #{index}"
-  ```
-
-* <a name="trailing-comma"></a>
```

Comment:
No trailing comma (ever).

Reply:
👎 💀

Reply:
I kinda agree with this, since the trailing comma may pop up in unexpected places that make it undesirable. See: https://github.com/uohzxela/ex_format/issues/90

Reply:
@josevalim, is it too late to suggest changing this rule?

I think omitting the trailing comma is mostly for aesthetics whereas keeping it is practical: With a trailing comma, it's going to be [a one line patch](https://dave.cheney.net/2014/10/04/that-trailing-comma) when you add something to the end of the list. Small patches are easier to review, produce less merge conflicts and—subjectively—encourage extension.